src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation-list/aggregation-list.component.scss

```diff
@@ -32,6 +32,19 @@
       justify-content: space-between;
       font-size: 14px;
       background-color: #eeeeee;
+      padding: 0 20px;
     }
   }
+
+  mat-toolbar {
```

Comment:
Move to app.

Reply:
we made a mistake on this. must stay on the ui lib